.mage/dev.go

```diff
@@ -68,8 +69,72 @@ func (Dev) Misspell() error {
 	)
 }
 
+var (
+	devDatabases          = []string{"cockroach", "redis"}
+	devDataDir            = ".dev/data"
+	devDatabaseName       = "ttn_lorawan_dev"
+	devDockerComposeFlags = []string{"-p", "lorawan-stack-dev"}
+)
+
+func dockerComposeFlags(args ...string) []string {
+	return append(devDockerComposeFlags, args...)
+}
+
+func execDC(args ...string) error {
+	_, err := sh.Exec(nil, os.Stdout, os.Stderr, "docker-compose", dockerComposeFlags(args...)...)
+	return err
+}
+
+// DBStart starts the databases of the development environment
+func (Dev) DBStart() error {
+	if mg.Verbose() {
+		fmt.Printf("Starting dev databases\n")
+	}
+	if err := execDC(append([]string{"up", "-d"}, devDatabases...)...); err != nil {
+		return err
+	}
+
+	return execDC("ps")
+}
+
+// DBStop stops the databases of the development environment
+func (Dev) DBStop() error {
+	if mg.Verbose() {
+		fmt.Printf("Stopping dev databases\n")
+	}
+	return execDC(append([]string{"stop"}, devDatabases...)...)
+}
+
+// DBErase erases the databases of the development environment
+func (Dev) DBErase() error {
+	mg.Deps(Dev.DBStop)
+	if mg.Verbose() {
+		fmt.Printf("Erasing dev databases\n")
+	}
+	return os.RemoveAll(devDataDir)
+}
+
+// DBSQL starts an SQL shell
+func (Dev) DBSQL() error {
+	mg.Deps(Dev.DBStart)
+	if mg.Verbose() {
+		fmt.Printf("Starting SQL shell\n")
+	}
+	return execDC("exec", "cockroach", "./cockroach", "sql", "--insecure", "-d", devDatabaseName)
+}
+
+// DBRedisCli starts a Redis-CLI shell
+func (Dev) DBRedisCli() error {
+	mg.Deps(Dev.DBStart)
+	if mg.Verbose() {
+		fmt.Printf("Starting Redis-CLI shell\n")
+	}
+	return execDC("exec", "redis", "redis-cli")
+}
+
 // InitStack initializes the Stack.
 func (Dev) InitStack() error {
+	mg.Deps(Dev.DBStart)
```

Comment:
Not everyone uses the databases in docker, so let's not force this.

Reply:
Followed previous behaviour, can remove. Will need to update DEVELOPMENT.md instructions accordingly